skyrim-platform/src/tilted/ui/DX11RenderHandler.h

```diff
@@ -2,17 +2,14 @@
 
 #include "MyRenderHandler.h"
 #include "TextToDraw.h"
+#include <DirectXTK/CommonStates.h>
+#include <DirectXTK/SpriteBatch.h>
+#include <DirectXTK/SpriteFont.h>
 #include <Signal.hpp>
 #include <functional>
 #include <mutex>
 #include <wrl.h>
 
-namespace DirectX {
-class SpriteBatch;
-class CommonStates;
-class SpriteFont;
-}
```

Comment:
`inline namespace DX11` appeared which came into conflict with this fwd declaration. Probably could also be fixed by adding it into this fwd declaration, but this would probably break older stuff...

Anyway, this file isn't included from anywhere else, so no much sense in making fwd declarations here